DESCRIPTION

```diff
@@ -48,7 +49,8 @@ Suggests:
     magrittr,
     webshot,
     spelling,
-    covr
+    covr,
+    ragg
```

Comment:
Why `ragg`? I don't see any usage

```suggestion
    covr
```

Reply:
Good catch.  I don't know why this is in here.  Fixing